torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -567,6 +567,7 @@ static void PrepareForRemoveMutations(MutationRemover& mr, Block* b) {
       if (it != node->inputs().end()) {
         int index = std::distance(node->inputs().begin(), it);
 
+        std::string input_name = input->debugName();
```

Comment:
Why is 'string input_name' variable needed? It looks it's not used anywhere.

Reply:
lines 583 and 598.